### PR TITLE
Add "Placeholder" withdraw option to the Menu Entry Swapper's "Bank W…

### PR DESCRIPTION
**Pull request by Chicagolight**
Add "Placeholder" withdraw option to the Menu Entry Swapper's "Bank Withdraw Shift-Click" menu.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 1]
```diff
@@ -540,7 +540,7 @@ public void onMenuEntryAdded(MenuEntryAdded menuEntryAdded)
 		// Deposit- op 1 is the current withdraw amount 1/5/10/x
 		if (shiftModifier() && config.bankWithdrawShiftClick() != ShiftWithdrawMode.OFF
 			&& menuEntryAdded.getType() == MenuAction.CC_OP.getId() && menuEntryAdded.getIdentifier() == 1
-			&& menuEntryAdded.getOption().startsWith("Withdraw"))
+			&& (menuEntryAdded.getOption().startsWith("Withdraw") || menuEntryAdded.getOption().equals("Placeholder")))
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java:543]: This is here to detect top-most op, which is added last, in order to do the swap once. `Placeholder` is never that (it is always the current withdraw mode). So I think you should be able to remove this?

Comment (Chicagolight) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: The file has been reverted.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/ShiftWithdrawMode.java [hunk 2]
```diff
@@ -40,6 +40,7 @@
 	// chambers of xeric storage units do not have an "all-but-1" option, so this option will choose "Withdraw-all"
 	// instead when using the storage unit.
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/ShiftWithdrawMode.java:41]:
```suggestion
	// chambers of xeric storage units do not have an "all-but-1" or a "placeholder" option, so these options will
	// choose "Withdraw-all" instead when using the storage unit.
```

Comment (Chicagolight) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/ShiftWithdrawMode.java]: Updated comments

**Comment by makingcomment** [comment]
Why does the comment say `choose "Withdraw-all"` twice?

**Pull request merged**